### ISSUE TITLE
Error: parsing Azure ID: parse "": empty url when no public IP is set on mgmt interface

### ISSUE THREAD

**Issue by JelleBroekhuijsen** [F5Networks/terraform-azure-bigip-module#51]
When variable mgmt_subnet_ids.public_ip == false:

```
│ Error: parsing Azure ID: parse "": empty url
│
│   with module.bigip[0].azurerm_network_interface.mgmt_nic[0],
│   on .terraform\modules\bigip\main.tf line 309, in resource "azurerm_network_interface" "mgmt_nic":
│  309:     public_ip_address_id          = local.bigip_map["mgmt_subnet_ids"][count.index]["public_ip"] ? azurerm_public_ip.mgmt_public_ip[count.index].id : ""

```

Can be resolved by replacing "" with null statement on line 309.

**Comment by KrithikaChidambaram** [comment]
Hi, tracking this internally at INFRAANO-1170, thanks!

**Comment by RavinderReddyF5** [comment]
fixed in v1.2.9
